source/Mlos.Python/mlos/Optimizers/RegressionModels/LassoCrossValidatedRegressionModel.py

```diff
@@ -89,6 +91,10 @@ def __init__(
         self.partial_hat_matrix_ = 0
         self.regressor_standard_error_ = 0
 
+        # THE HACK

```

Comment:
We may need to explain a little more here. If I remember right:

When LassoCV is used as part of RERF, it cannot reasonably compute the upper and lower bounds on its input space dimensions, as they are a polynomial combination of inputs to RERF. Thus, it approximates them with the empirical min and max. These approximations are biased: the lower bound is too large, the upper bound is too small. Consequently, during scoring, LassoCV is likely to see input outside of these bounds, but we still want LassoCV to produce predictions for those points. So we introduce a little hack: whenever LassoCV is instantiated as part of RERF, it should skip input filtering on predict. This field, controls this behavior.

Feel free to just copy-paste that in, or polish it to your liking!